pycbc/tmpltbank/bank_output_utils.py

```diff
@@ -299,3 +299,50 @@ def output_sngl_inspiral_table(outputFile, tempBank, metricParams,
 
     # write the xml doc to disk
     ligolw_utils.write_filename(outdoc, outputFile)
+
+
+def output_bank_to_hdf(outputFile, tempBank, programName="", optDict=None):
```

Comment:
If possible, it would be good to use this:
https://github.com/gwastro/pycbc/blob/master/pycbc/waveform/bank.py#L373
to avoid duplicating all this.

Reply:
That looks like it may not be possible - the TemplateBank object itself uses a file to read in.

Unless someone wants to completely rewrite all the bank codes to unify them!